packages/application-shell/src/components/login/messages.js

```diff
@@ -45,17 +45,13 @@ export default defineMessages({
       'Text for the link to the legacy MC when the user has no projects',
     defaultMessage: 'set up a project in the current Merchant Center',
   },
-  forgotPassowordTitle: {
-    id: 'LoginForm.forgotPassowordTitle',
+  forgotPasswordTitle: {
+    id: 'LoginForm.forgotPasswordTitle',
     description:
       'An informative message for redirecting the user to ' +
-      'forgot password page in the legacy MC',
-    defaultMessage: 'We are redirecting you to the Forgot Password form.',
-  },
-  forgotPassowordSubtitle: {
-    id: 'LoginForm.forgotPassowordSubtitle',
-    description: 'Countdown waiting for redirect',
-    defaultMessage: 'Please wait. {countdown}',
+      'forgot password page in the admin center',
+    defaultMessage:
+      'We are redirecting you to the Forgot Password form in {countdown} seconds.',
```

Comment:
nitpick: pleaaase to help me stay sane, don't capitalize non proper nouns 😱 🇩🇪

Reply:
Don't look at me, I didn't translate this in the first place 😇